bundle/jsky.app.ot/src/main/java/jsky/app/ot/tpe/feat/TpeGuidePosFeature.java

```diff
@@ -466,17 +468,21 @@ public void draw(Graphics g, TpeImageInfo tii) {
 
     /**
      */
-    public Option<Object> dragStart(TpeMouseEvent tme, TpeImageInfo tii) {
-        TargetObsComp obsComp = getTargetObsComp();
+    public Option<Object> dragStart(final TpeMouseEvent tme, final TpeImageInfo tii) {
+        final TargetObsComp obsComp = getTargetObsComp();
         if (obsComp == null) return None.instance();
 
-        TargetEnvironment env = obsComp.getTargetEnvironment();
+        final TargetEnvironment env = obsComp.getTargetEnvironment();
+
+        // We do not allow deletion of auto guide stars.
+        final GuideGroup        gp  = env.getOrCreatePrimaryGuideGroup();
```

Comment:
The idea is that there will always be an automatic guide group. It doesn't have to be the primary guide group, but there will be one in every observation.